chain-impl-mockchain/src/header/cstruct.rs

```diff
@@ -20,8 +19,7 @@ pub(super) type BftLeaderId = [u8; 32];
 pub(super) type BftSignature = [u8; 64];
 
 pub(super) type GpNodeId = [u8; 32];
-pub(super) type GpVrfProof =
-    [u8; <EllipticCurve2hashDh as VerifiableRandomFunction>::VerifiedRandomOutput::BYTES_LEN];
+pub(super) type GpVrfProof = [u8; 96];
```

Comment:
I would keep using the constant of the VRF proof. If we ever decide to implement the VRF as defined in the IETF standard, this size would change.

Reply:
The thing is that no other constant in that file is bound to anything, so I thought to remove it not to break consistency, but either way is fine for me.